### PR TITLE
Add several spelling corrections

### PR DESCRIPTION
**Pull request by luzpaz**

### CODE REVIEW

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -8792,6 +8793,18 @@ fcound->found
 feasabile->feasible
 feasability->feasibility
 feasable->feasible
+featch->fetch
+featchd->fetched
+featched->fetched
+featcher->fetcher
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt]: Suggest `feather, feature, fetcher`

File: codespell_lib/data/dictionary.txt [hunk 2]
```diff
@@ -13998,6 +14011,7 @@ ojects->objects
 ojekts->objects
 okat->okay
 oldes->oldest
+oll->all, ole, old,
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt]: Also `Olly, oil`

File: codespell_lib/data/dictionary.txt [hunk 3]
```diff
@@ -15459,6 +15473,8 @@ preocupation->preoccupation
 preoperty->property
 prepair->prepare
 prepaired->prepared
+preparetion->preparation
+preparetions->preparation
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt]: This should be the plural suggestion.

File: codespell_lib/data/dictionary.txt (hunk 1 above)
Comment (luzpaz) [reply codespell_lib/data/dictionary.txt]: 1f739d0

File: codespell_lib/data/dictionary.txt (hunk 2 above)
Comment (luzpaz) [reply codespell_lib/data/dictionary.txt]: 1f739d0

File: codespell_lib/data/dictionary.txt (hunk 3 above)
Comment (luzpaz) [reply codespell_lib/data/dictionary.txt]: 4991924

File: codespell_lib/data/dictionary.txt [hunk 4]
```diff
@@ -2147,6 +2147,7 @@ auospacing->autospacing
 auot->auto
 auotmatic->automatic
 auromated->automated
+aussian->gaussian
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt]: Suggest Russian

File: codespell_lib/data/dictionary.txt [hunk 5]
```diff
@@ -8798,6 +8799,18 @@ fcound->found
 feasabile->feasible
 feasability->feasibility
 feasable->feasible
+featch->fetch
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt]: Suggest reach

File: codespell_lib/data/dictionary.txt (hunk 4 above)
Comment (luzpaz) [reply codespell_lib/data/dictionary.txt]: 70bc265

File: codespell_lib/data/dictionary.txt (hunk 5 above)
Comment (luzpaz) [reply codespell_lib/data/dictionary.txt]: 70bc265

**Pull request merged**